### PR TITLE
Removes Pranksimov from unique AI station trait

### PR DESCRIPTION
**Pull request by Coolrune206**
## What Does This PR Do

Removes the Pranksimov lawset from the Unique AI station trait - it can no longer roll at roundstart.

## Why It's Good For The Game

Look, I'm sad that I have to make this PR, I fundamentally love the concept of Pranksimov, and was super psyched to see it used more. However, the problem is that humor is very much subjective, and when you consider that the lawset only has restrictions "unless it is funny" when really anything can be seen as "funny", this is pretty much the same as giving the station an unlawed AI (as it can ignore its laws for funny) that CC has disallowed them to relaw, to put up with for most of the shift.

I really do think Pranksimov is a funny lawset when used correctly, and I might look into adding it as an alternative form of Ion Storm ion law event, but it makes absolutely zero sense for the context it is used in as an experimental AI law that CC is testing - unlike the other options which are fundamentally logical lawsets or those that would benefit from experimentation, this one really is just included for the funny and makes no sense lore-wise.

Removing it here will put it back to being an option to Traitor clowns to subvert the AI (that they absolutely should consider) by c-magging a crewsimov board, which I think will add a little bit of rarity and freshness to this lawset.

## Testing

Confirmed it built. Honestly, no clue if I even can test Station Traits on my local instance as you can only modify things for the following round and I don't have a database, so I think each round I start is treated like a fresh boot. However, the nature of the change should indicate no further testing is necessary as I'm simply disabling it from selecting.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>
image
_explanation I gave above, you don't need to see it twice_
image

## Changelog

:cl:
tweak: Unique AI Station Trait no longer rolls Pranksimov
/:cl:

### CODE REVIEW

File: code/datums/ai_law_sets.dm [hunk 1]
```diff
@@ -145,7 +145,6 @@
 	name = "Pranksimov"
 	law_header = "Comedy Routine"
 	selectable = TRUE
-	unique_ai = TRUE //honk
```
Comment (AffectedArc07) [inline code/datums/ai_law_sets.dm:148]: [OOS] 

This var name offends me. `applicable_for_unique_ai_station_trait`, while verbose, would make WAY more sense for this as opposed to `unique_ai`.

Comment (Coolrune206) [reply code/datums/ai_law_sets.dm:148]: OOS meanin' outside of scope?

Comment (AffectedArc07) [reply code/datums/ai_law_sets.dm:148]: Yeah

**Review by lewcc — APPROVED** [review]
Code change lgtm

**Comment by MLGTASTICa** [comment]
Me when the RD can't be bothered to take the 15 seconds necesarry to build a hidden upload and reset the AI.

**Review by Sirryan2002 — APPROVED** [review]

**Review by 1080pCat — APPROVED** [review]
icky light mode user 😔

**Comment by BiancaWilkson** [comment]
> Removing it here will put it back to being an option to Traitor clowns to subvert the AI (that they absolutely should consider) by c-magging a crewsimov borg, which I think will add a little bit of rarity and freshness to this lawset.
Cmagging cyborgs current doesn't do anything

**Comment by Coolrune206** [comment]
> > Removing it here will put it back to being an option to Traitor clowns to subvert the AI (that they absolutely should consider) by c-magging a crewsimov borg, which I think will add a little bit of rarity and freshness to this lawset.
> 
> Cmagging cyborgs currently doesn't do anything

Ah yeah I meant to say "crewismov board", brain was fried when I wrote this 💀

**Comment by Komrad822** [comment]
Player issue, pranksimov=/= antagonist. Command is always able to fax CC for permission to change the AI module during that trait. The same can be said said about any other 'controversial' law set that. What I mean by this is many lawsets can be interpreted in a way that is negative to crew by the AI and thus its up to players to ahelp the AI or admins step in and speak to them. We should not be removing lawsets from the list just because a small amount of people interpret it in bad faith.

**Comment by Coolrune206** [comment]
> Player issue, pranksimov=/= antagonist. Command is always able to fax CC for permission to change the AI module during that trait. The same can be said said about any other 'controversial' law set. What I mean by this is many lawsets can be interpreted in a way that is negative to crew by the AI and thus its up to players to ahelp the AI or admins step in and speak to them. We should not be removing lawsets from the list just because a small amount of people interpret it in bad faith.

Speaking as an admin, Pranksimov is consistently the one people _do_ have to ask to remove, if they know they're allowed to at all.
Speaking from a logical standpoint, even the other "controversial" lawsets such as Tyrant and Peacekeeper have valid uses and would make logical sense for Central Command to test. Pranksimov was explicitly added as a lawset available to antagonist Clowns who bought a Cmag to upload to the AI, hence why it is so directly (and deliberately) problematic.

**Comment by S34NW** [comment]
This PR has unfortunately failed to pass the relevant team vote, with 1 for and 2 against.
Some of the reasons for objection include (anonymised):

- I personally do not see much of a problem with Pranksimov from a Design standpoint, if it were something that the Admins discussed in Staff channels to remove due to rulebreaks and the PR was done off of that than I'd more amenable to removal.
- agreeing with the general sentiment here; if ais are being shitty with this lawset, it's up to Command to get it changed, or an ahelp to be made if it's especially bad. the lawset itself is alright, and doesn't need removal

Below is by a team member who abstained:
- Kinda neutral on the argument myself. Its funny and lets AI players have a bit of shenangains. But it is not really fitting IC wise. Unsure on this.

Please feel free to discuss this with the relevant team and see if you can rework it into something acceptable to them.

**Pull request closed without merging**